datasrc/seven/compile.py

```diff
@@ -261,7 +261,10 @@ class CNetObjHandler
 		lines += ['\t{']
 
 		for item in network.Objects:
-			for line in item.emit_validate():
+			base_item = None
+			if item.base:
+				base_item = next(i for i in network.Objects if i.name == item.base)
```

Comment:
```suggestion
				base_item = only([i for i in network.Objects if i.name == item.base])
```
with
```py
def only(x):
    if len(x) != 1:
        raise ValueError
    return list(x)[0]
```
to capture intent.